frontend/public/components/factory/table.tsx

```diff
@@ -338,11 +338,11 @@ export type RowFunctionArgs<T = any, C = any> = {
   customData?: C;
 };
 
-export type RowFunction<T = any, C = any> = (args: RowFunctionArgs<T, C>) => JSX.Element;
+export type RowFunction<T = any, C = any> = (args: RowFunctionArgs<T, C>) => React.ReactElement;
```

Comment:
@christianvogt FYI, returning `ReactNode` [as suggested](https://github.com/openshift/console/pull/4783#discussion_r397462973) yields some TS errors, so I've changed it to `ReactElement`.